spec/views/posts/show.html.erb_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'posts #show', type: :feature do
+  before :each do
+    @user = User.create(
+      name: 'Benson',
+      photo: 'https://www.pexels.com/photo/cottages-in-the-middle-of-beach-753626/',
+      bio: "I'm finding it hard coping with ruby on rails",
+      posts_counter: 0
+    )
+    Post.create(author: @user, title: 'Title', text: 'Text', comments_counter: 0, likes_counter: 0)
+    Post.create(author: @user, title: 'Title2', text: 'Text2', comments_counter: 0, likes_counter: 0)
+    Post.create(author: @user, title: 'Title3', text: 'Text3', comments_counter: 0, likes_counter: 0)
+    @post = Post.create(author: @user, title: 'Title4', text: 'Text4', comments_counter: 0, likes_counter: 0)
+    User.create(
+      name: '1st',
+      photo: 'https://www.pexels.com/photo/cottages-in-the-middle-of-beach-753626/',
+      bio: "I'm finding it hard coping with ruby on rails",
+      posts_counter: 0
+    )
+    visit "/users/#{@user.id}/posts/#{@post.id}"
+  end
+
+  describe 'post show page' do
+    it 'shows the post\'s title' do
+      expect(page).to have_content(@post.title)
+    end
+
+    it 'shows who wrote the post' do
+      expect(page).to have_content(@user.name)
+    end
+
+    it 'shows how many comments there are on a post' do
+      expect(page).to have_content(@post.comments.count)
+    end
+
+    it 'shows how many likes there are on a post' do
+      expect(page).to have_content(@post.likes.count)
+    end
+
+    it 'shows the post body' do
+      expect(page).to have_content(@post.text)
+    end
+
+    it 'shows all comments and the author name' do
+      comments = @post.comments
+      comments.each do |comment|
+        expect(page).to have_content(comment.author.name)
+        expect(page).to have_content(comment.text)
+      end
+    end
```

Comment:
- [ ] Kindly make sure all your tests are passing as required for the project. From the image below you can the result with with the list of failed tests of your `views`(`rspec spec/views`)
![Screenshot from 2022-11-02 02-02-04](https://user-images.githubusercontent.com/83042742/199364588-24d58d73-56a2-4c5c-ab97-40f6d24e1276.png)

Reply:
@tresorsawasawa , All tests are now passing